SparkPost/2web2ui
# AC-921: Billling Page should allow manually billed customers to switch to self-serve

### What Changed
 - Manually billed banner brings to credit card addition form
 - Form transitions customer into self serve billing

### How To Test
 - Create a new account (w/o selecting plan)
 - Transition to a manually billed customer (check the admin docs for billing API)
 - Go to /account/billing
   -Verify a manually billed banner exists
- Check that the link navigates to /account/billing/enable-automatic
 - Check that submission works and navigates back to /account/billing
 - Should now show billing summary rather than manually billed banner

### To Do
- [x] Wait for CORS fix in API
- [x] Wait for fix in delay of flag
- [x] Update check for manually billed to be in billing/subscription

### Notes
Waiting for update in API for flag change
